### ISSUE TITLE
[Windows] multiple null-check exceptions after controller dispose

### ISSUE THREAD

**Issue by nikitatg** [zezo357/flutter_meedu_videoplayer#4]
Greetings! 
I began to use package on Windows and noticed that after closing the player (and disposing player controller) flutter debug console receiving multiple
`currentVolume Null check operator used on a null value`
The reason i found at controller.dart#1025
```dart
        _timerForGettingVolume?.cancel();
        _timerForGettingVolume =
            Timer.periodic(Duration(milliseconds: 250), (Timer t) async {
--->>      _timerForGettingVolume = null;
...
```
Loosing link to timer makes it stay in memory and continue ticking. And when disposing a controller we can't stop it because link was lost. Locally i've commented this line and exception diappeared; but i'm not quite shure what's going on at that method/timer and that's why i can't pull-request such "fix".

Thank you for useful package.

**Comment by zezo357** [comment]
I will check on it again

Thank you for the explaining ❤️

**Comment by zezo357** [comment]
fixed in 2.0.2 

sorry for late update

feel free to open an issue if something is not working as expected
